src/assets/scripts/client/airline/AirlineCollection.js

```diff
@@ -32,6 +34,21 @@ export default class AirlineCollection extends BaseCollection {
     }
 
     /**
+     * List of all `flightNumbersInUse`
+     *
+     * Used when generating new flightNumbers to verify a new
+     * number isn't already in use
+     *
+     * @property flightNumbers
+     * @return {array<string>}
+     */
+    get flightNumbers() {
+        const flightNumberList = _map(this._items, (airline) => airline.flightNumbers);
+
+        return _flatten(flightNumberList);
+    }
```

Comment:
TBH, I'm not a fan of the additional _flatten here. This may still change to a `...`.

As far as speed goes, I doubt it.  At least with the data loads this property would be seeing.  100, maybe 200?  I'd be concerned about performance here once we got into the 5k range, though, realistically I could see this being fine up to 100k range.  There isn't much logic here, the `_flatten` part is the most intense.